www/js/survey/enketo/enketo-notes-list.js

```diff
@@ -33,7 +33,8 @@ angular.module('emission.survey.enketo.notes-list',
     }
 
     $scope.setDisplayTime = function(entry) {
-      const timezone = $scope.timezone;
+      const timezone = $scope.timelineEntry.start_local_dt?.timezone
```

Comment:
To confirm: We are doing this because if we were to use `$scope.timezone`, the display time would be in the device's local time rather than the local time to the TimelineEntry (i.e. if the TimelineEntry was initially tracked in Hawaii but the user is now in Colorado, the display time will still be in Hawaii time)?

Is `start_local_dt` the best option for determining time zone? I believe this is good because in testing with Hawaii TimelineEntries I haven't seen any issues, but I just want to get input on this.

Reply:
This is the same value that $scope.timzone had before, but I removed the scope variable because it was redundant to have as a scope variable when we can just grab the timezone from the timlineEntry

But yes, the overall reason this is necessary is that we want the timestamps of 'notes' to show up in the timezone of the trip/place they belong to.
The only reason using `start_local_dt` could be misleading is for trips that go across timezones. Not sure if we want to dig into that (fairly uncommon) scenario right now.

When the survey is filled out, we assume that the times entered are in the same timezone. So I think it's fair to assume that we will also display them as such, according to the starting timezone.